formats/financial_release/publisher/links.json

```diff
@@ -0,0 +1,13 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "type": "object",
+  "additionalProperties": false,
+  "required": [
+    "campaign_page"
+  ],
+  "properties": {
+    "campaign_page": {
```

Comment:
we will go ahead with removing the campaign_page link for now, as it is not necessary at the moment.